draft-ietf-scitt-architecture.md

```diff
@@ -175,6 +177,158 @@ SCITT achieves this by having producers publish information in a Transparency Se
+Finally, a third update includes a new release (updated version) of the formerly insecure software component.
+For this release, both the software product and the affected software component are deemed secure by the producer and consumers.
+
+A consumer of a released software wants to:
+
+* know where to get these security statements from producers and third-parties related to the software product in a timely and unambiguous fashion
+* attribute them to an authoritative issuer
+* associate the statements in a meaningful manner via a set of well-known semantic relationships
+* consistently, efficiently, and homogeneously check their authenticity
+
+Without SCITT there would be no standardized way to:
+
+* know the various sources of statements
+* express the provenance and historicity of statements
+* relate and link various heterogeneous statements in a simple fashion
+* check that the statement comes from a source with authority to issue that statement
+
+### Promotion of a Software Component by Multiple Entities
+
+A software component (e.g., a library) released by a certain original producer is becoming popular.
+The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
+Over time, due to its enhanced applicability to various products, there has been an increasing amount of multiple providers of the same software component version on the internet.
+
+Some providers include this particular software component as part of their release package bundle and provide the package with proof of authenticity using their own issuer authority.
+Some packages include the original statement of authenticity, and some do not.
+Over time, some providers no longer offer the exact same software component source code but pre-compiled software component binaries.
+Some sources do not provide the exact same software component, but include patches and fixes produced by third-parties, as these emerge faster than solutions from the original producer.
+Due to complex distribution and promotion life-cycle scenarios, the original software component takes myriad forms.
+
+A consumer of a released software wants to:
+
+* understand if a particular provider is actually the original provider or a promoter
+* know if and how the source, or resulting binary, of a promoted software component differs from the original software component
+* check the provenance and history of a software component's source back to its origin
+* assess whether to trust a promoter or not
+
+Without SCITT there would be no standardized way to:
+
+* reliably discern if a provider is the original producer or is a trustworthy promoter or is an illegitimate provider
+* track the provenance path from an original producer to a particular provider
+* check the trustworthiness of a provider
+* check the integrity of modifications or transformations applied by a provider
+
+### Software Integrator Assembling a Software Product for a Smart Car
```

Comment:
```suggestion
### Software Integrator Assembling a Software Product for an autonomous vehicle
```

Smart Car is a brnad name. Better avoided.

Reply:
perhaps just "connected", but the example works either way